feature/platform/transceiver/zr_low_power_mode_test/README.md

```diff
@@ -0,0 +1,114 @@
+# TRANSCEIVER-13: Configuration: 400ZR Transceiver Low Power Mode Setting.
+
+## Summary
+
+Validate 400ZR transceiver is able to move to low power consumption mode when
+the interface/config/enabled state is set to "False"
+
+**NOTE:** The Module Power Mode dictates the maximum electrical power that the
+module is permitted to consume while operating in that Module Power Mode.
+The Module Power Mode is a function of the state of the Module State Machine.
+Two Module Power Modes are defined:
+  * In Low Power Mode (characteristic of all MSM steady states except
+    ModuleReady) the maximum module power consumption is defined in the form
+    factor-specific hardware specification.
+  * In High Power Mode (characteristic of the MSM state ModuleReady) the
+    implementation dependent maximum module power consumption is advertised in
+    the MaxPower Byte 00h:201. More details in the CMIS link below.
+
+Link to CMIS:
+https://www.oiforum.com/wp-content/uploads/CMIS5p0_Third_Party_Spec.pdf
+
+## Procedure
+
+*   Connect two ZR optics using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module.
+*   To establish a point to point ZR link ensure the following:
+      * Both transceivers state is enabled.
+      * Both transceivers are set to a valid target TX output power
+        example -10 dBm.
+      * Both transceivers are tuned to a valid centre frequency
+        example 193.1 THz.
+## Testbed Type
+*   Typical test setup for this test is a DUT1 with 2 ports to 2 ATE ports or 2
+    ports to a second DUT2. For most tests this setup should be sufficient.
+    Ref: [Typical ATE<>DUT Test bed](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+*   A and Z ends of  the link should have same 400ZR PMD. For this test a
+    single DUT ZR port connected to a single ZR ATE port is also sufficient. 
+
+Once the ZR link is estabished proceed with the following:
+*   Verify that the following ZR transceiver OC path when set to False is able
+    to move to the low power mode as defined in the CMIS.
+    
+    *   /interfaces/interface/config/enabled
+
+*   In low power mode the module's Management interface should be available,
```

Comment:
optional, but are we going too deep into CMIS jargon here when this is an OpenConfig test? Could we essentially delete this bullet and the following one without losing any intent of this test?